src/StatesZoo/zalm_pair/zalm_pair.jl

```diff
@@ -0,0 +1,588 @@
+    -1)+eD)*(4*eD*(1+(-2)*Pd)*Pd+2*Pd^2+eD^2*(1+(-6)*
+    Pd+7*Pd^2))+c1^2*(2*eD*(3+(-7)*Pd)*Pd+4*Pd^2+
+    eD^2*(1+(-8)*Pd+11*Pd^2)))),0,0,0,0,0]
+    
+    v32 = [0,0,0,0,0,0,0,0,
+    0,0,0,0,0,0,0,0,0,0,0,0,0,0,(1/2)*2^(-1/2)*c0^3*c1^4*
+    c2*eAs^(1/2)*(eAs^2)^(1/2)*eBs^(3/2)*((-1)+eD)*eD*
+    ((-1)+Pd)^3*((-2)*Pd+eD*((-1)+3*Pd)),0,0,0,0,(1/2)*
+    2^(-1/2)*c0^2*c1^4*c2*eAs*(eAs^2)^(1/2)*eBs^(
+    1/2)*((-1)*c0*eBs^(1/2)+4*c2*(1+(-1)*eBs)^(1/2)*((
+    -1)*((-1)+eBs)*eBs)^(1/2)*((-1)+eD))*((-1)+eD)*eD^2*
+    ((-1)+Pd)^4,0,0,0,(-1/4)*c0^2*c2^2*eAs^2*eBs*((-1)+
+    eD)*((-1)+Pd)^2*(c1^4*((-1)+eBs)*((-1)+eD)*(8*eD*(
+    1+(-2)*Pd)*Pd+4*Pd^2+eD^2*(3+(-14)*Pd+15*Pd^2))+2*
+    c0^2*((-1)*Pd+eD*((-1)+2*Pd))*(4*c2^2*((-1)+eBs)*(
+    (-1)+eD)*((-1)*Pd+eD*((-1)+2*Pd))+c1^2*((-2)*Pd+eD*(
+    (-1)+3*Pd)))),0,0,0,0]
+    
+    v33 = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+    0,0,0,0,0,0,0,0,0,0,(1/2)*c0^2*c1^4*c2^2*eAs*(
+    eAs^2)^(1/2)*eBs*(eBs^2)^(1/2)*((-1)+eD)^2*eD^2*(
+    (-1)+Pd)^4,0,0,0,c0^4*c2^4*eAs^2*eBs^2*(((-1)+eD)
+    ^2)*((-1)+Pd)^2*(eD+Pd+(-2)*eD*Pd)^2,0,0,0]
+    
+    v34 = [0,0,0,0,0,
+    0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,c0^4*c1^2*c2^2*
+    eAs^(1/2)*(eAs^2)^(1/2)*eBs^(1/2)*(eBs^2)^(1/2)*((
+    -1)+eD)^2*eD*((-1)+Pd)^3*Pd,0,0,0,0,0,0,0,0,0,(-1/4)*
+    c0^2*c2^2*eAs^2*eBs*((-1)+eD)^2*((-1)+Pd)^2*(8*
+    c0^2*c2^2*((-1)+eBs)*((-1)+eD)*Pd*((-1)*Pd+eD*((-2)
+    +3*Pd))+2*c0^2*c1^2*Pd*((-2)*Pd+eD*((-3)+5*Pd))+
+    c1^4*((-1)+eBs)*(8*eD*(1+(-2)*Pd)*Pd+4*Pd^2+eD^2*
+    (3+(-14)*Pd+15*Pd^2))),0,0]
+    
+    v35 = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+    0,0,0,0,0,0,0,0,0,0,0,0,0,0,(1/2)*c0^2*c1^4*c2^2*
+    eAs*(eAs^2)^(1/2)*eBs*(eBs^2)^(1/2)*((-1)+eD)^2*
+    eD^2*((-1)+Pd)^4,0,0,0,0,(1/4)*c0^2*c1^4*c2^2*
+    eAs^2*eBs^2*((-1)+eD)^2*((-1)+Pd)^2*(8*eD*(1+(-2)
+    *Pd)*Pd+4*Pd^2+eD^2*(3+(-14)*Pd+15*Pd^2)),0]
+    
+    v36 = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+    0,0,c0^4*c2^4*eAs^2*eBs^2*((-1)+eD)^3*(((-1)+Pd)
+    ^2)*Pd*((-1)*Pd+eD*((-2)+3*Pd))]
+
+    hcat(v1, v2, v3, v4, v5, v6, v7, v8, v9, v10, v11, v12, v13, v14, v15, v16, v17,
+    v18, v19, v20, v21, v22, v23, v24, v25, v26, v27, v28, v29, v30, v31, v32, v33, v34, v35, v36)'
+end
+
+function cascaded_source_spin(Ns,gA,gB,eAm,eBm,eAs,eBs,eD,Pd,Pdo1,Pdo2,VisF)
+    m11=ret_c11(Ns,gA,gB,eAm,eBm,eAs,eBs,eD,Pd,Pdo1,Pdo2,VisF)
+    m22=ret_c22(Ns,gA,gB,eAm,eBm,eAs,eBs,eD,Pd,Pdo1,Pdo2,VisF)
+    m33=ret_c33(Ns,gA,gB,eAm,eBm,eAs,eBs,eD,Pd,Pdo1,Pdo2,VisF)
+    m44=ret_c44(Ns,gA,gB,eAm,eBm,eAs,eBs,eD,Pd,Pdo1,Pdo2,VisF)
+    m23=ret_c23(Ns,gA,gB,eAm,eBm,eAs,eBs,eD,Pd,Pdo1,Pdo2,VisF)
+    
+    return [m11 0 0 0 ; 0 m22 m23 0 ; 0 m23' m33 0 ; 0 0 0 m44]
+end
+
+
+"""
+Generate symbolic object for the spin-spin density matrix for a 
+cascaded source swapped with emissive spin memories from the paper [prajit2023entangling](@cite)
+"""
+@withmetadata struct ZALMpair <: AbstractTwoQubitState
+    Ns::Float64
```

Comment:
Could you document what each of these parameters means? Prajit has that explained in his readmes -- basically just copy that, for all symbolic structs.